### DISCUSSION TITLE
Node Graph for service graph in a dashboard

### DISCUSSION THREAD

**Discussion by Symbianx** [grafana/grafana#57150] [Dashboards, Panels & Visualizations]
Hello everyone 👋 
I'm trying to get a Node Graph visualization in a dashboard, similar to the Service Graph that we get in the Tempo Explore tab.

I'm trying to use prometheus as a data source for the NodeGraph but I can't seem to get it working.

I saw the docs aroud [NodeGraph](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/node-graph/) and tried to implement the 2 data frames using different promql queries but I'm not sure if that's the way to go since it didn't work and kept complaining of missing fields.

The metrics exist in prometheus since the same data source is used to display the service graph in the Explore tab.

Is this a supported use case and if so how do I get it to work?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
